src/main/kotlin/fi/metatavu/metaform/server/controllers/MetaformKeycloakController.kt

```diff
@@ -806,6 +971,23 @@ class KeycloakController {
         ).firstOrNull()
     }
 
+    /**
+     * Translates REST User to UserRepresentation
+     *
+     * @param user User
+     * @return UserRepresentation
+     */
+    private fun translateUserToUserRepresentation(user: User): fi.metatavu.metaform.keycloak.client.models.UserRepresentation {
```

Comment:
This should be moved to separate translator

Reply:
Done